src/main/java/org/entando/entando/aps/system/services/label/LabelService.java

```diff
@@ -133,9 +126,9 @@ public LabelDto addLabelGroup(LabelDto labelRequest) {
     @Override
     public void removeLabelGroup(String code) {
         try {
-            ApsProperties labelGroup = this.getI18nManager().getLabelGroup(code);
-            if (null == labelGroup) {
-                logger.warn("no label found with key {}", code);
+            if (mustGetLabelGroup(code, false) == null) {
+                // TODO: $$$ CONSIDER THROWING EXCEPTION INSTEAD
+                // ... (see other action and other services: like LanguageService)
                 return;
             }
```

Comment:
@eugeniosant please check

the other actions throws an exception in case of label group not found
Also LanguageService, which is specular, throws an exception in this scenario.

Reply:
I think a Resource Not Found Exception should be thrown as in other cases. For me it's a mistake to leave only a warning

Reply:
Thanks.
Since this PR changed scope I'll fix it in another PR.